acceptance/setup/aio/pre-suite/020_AIO_Workarounds.rb

```diff
@@ -1,10 +1,3 @@
-step "(PUP-4004) Set permissions on puppetserver directories that currently live in the agent cache dir"
-%w[reports server_data yaml bucket].each do |dir|
-  on master, "install --directory /opt/puppetlabs/puppet/cache/#{dir}"
-end
-on master, "chown -R puppet:puppet /opt/puppetlabs/puppet/cache"
-on master, "chmod -R 750 /opt/puppetlabs/puppet/cache"
-
```

Comment:
@er0ck can you undo this change (as it's already been done in https://github.com/puppetlabs/puppet/commit/0c138daad9ba0ed595956a2ba046d0cab08756ef), and update the commit message?